src/main/resources/hocs-migration-schema.json

```diff
@@ -50,15 +50,71 @@
           }
         ]
       }
+    },
+    "correspondent" : {
+      "description": "A single correspondent",
+      "type": "object",
+      "properties": {
+        "fullName": {
+          "type": "string"
+        },
+        "correspondentType":  {
+          "type": "string"
+        },
+        "address1":  {
+          "type": "string"
+        },
+        "address2": {
+          "type": "string"
+        },
+        "address3": {
+          "type": "string"
+        },
+        "postcode": {
+          "type": "string"
+        },
+        "country": {
+          "type": "string"
+        },
+        "organisation": {
+          "type": "string"
+        },
+        "telephone": {
+          "type": "string"
+        },
+        "email": {
+          "type": "string"
+        },
+        "reference": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "fullName",
+        "correspondentType"
+      ]
+    },
+    "otherCorrespondents": {
+      "description": "Other correspondents",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/correspondent"
+      }
+    },
+    "primaryCorrespondent": {
```

Comment:
Doesn't look like you're using this definition, because the primaryCorrespondent is a correspondent type

Reply:
This has been removed after our schema pairing session.